src/core/frameworks/detect.ts

```diff
@@ -319,7 +319,7 @@ function orderFrameworksByParent(detectedFolders: DetectedFolder[]): void {
 }
 
 function findAllFiles(fileglob: string, files: string[]): string[] {
-  const { regex } = globrex(`?(*/)${fileglob}`, { extended: true, flags: "i" } as globrex.Options);
+  const { regex } = globrex(`?(*${path.sep})${fileglob}`, { extended: true, flags: "i" } as globrex.Options);
```

Comment:
Why not use the existing `globalyzer` module instead of `globrex`?

Reply:
TODO: remove globalyzer.

Reply:
done